astrogrism/config/HST/create_HST_reffiles.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+
+from astrogrism.config.HST.reference_file_generators._generate_specwcs import create_grism_specwcs  # noqa
+from astrogrism.config.HST.reference_file_generators._generate_wavelengthrange import create_tsgrism_wavelengthrange  # noqa
+# from astrogrism.config.HST.reference_file_generators.reference_file_generators._generate_distortion import create_distortion  # noqa
+
+
+def create_reference_files(conffile, hst_grism, outpath=Path.cwd()):
+    create_tsgrism_wavelengthrange(outname=str(Path(outpath) / "HST_wavelengthrange.asdf"))
```

Comment:
We probably don't want every file generated by this script to have the same name. Some other options would be to take output directory and output filename as separate inputs, or to generate the filename based on the `conffile` string. I know the automatic name generation I wrote in the specwcs file (`outname = "WFC3_{}_specwcs.asdf".format(pupil)`) ended up being insufficient (ACS isn't WFC3!) so I put in the filename manually for ACS.

Reply:
Good idea; I've implemented an `outbasename` argument that the user can customize. Rather than having three separate arguments for each reference file, I appended the reference file type to the end of the shared basename. Take a look and let me know what you think!